offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/projectcreation/CreateProjectView.groovy

```diff
@@ -40,10 +40,10 @@ class CreateProjectView extends VerticalLayout{
     static {
         spaceSelectionActionText.put(
                 CreateProjectViewModel.SPACE_SELECTION.EXISTING_SPACE,
-                "an existing project space")
+                "An existing project space")
         spaceSelectionActionText.put(
                 CreateProjectViewModel.SPACE_SELECTION.NEW_SPACE,
-                "a new project space")
+                "A new project space")
```

Comment:
would not change that. As far as I can see this completes the sentence and should be in lowe case (its not a heading)

![Screenshot 2021-04-07 at 09 00 43](https://user-images.githubusercontent.com/20778777/113824178-bf3cb680-977f-11eb-9a97-23c4b388b1a3.png)

Reply:
Alrighty